src/_data/site.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "FLOE Project",
+    "shortDesc": "Flexible Learning for Open Education",
+    "url": "https://www.floeproject.org",
+    "authorName": "OCAD University",
+    "authorWebsite": "https://www.ocadu.ca/",
```

Comment:
Author Name should be `Inclusive Design Research Centre, OCAD University`
Author website should be `https://idrc.ocadu.ca`

Reply:
fixed